packages/ast-spec/tests/util/parsers/babel.ts

```diff
@@ -0,0 +1,50 @@
+import { parse, ParserPlugin } from '@babel/eslint-parser';
+import { Fixture, ParserResponse } from './parser-types';
+
+const PLUGINS: ParserPlugin[] = [
+  // TODO - enable classFeatures instead of classProperties when we support it
+  // 'classFeatures',
+  'classProperties',
+  'decorators-legacy',
+  'importAssertions',
+  'typescript',
+];
+
+export function parseBabel(fixture: Fixture, contents: string): ParserResponse {
+  const plugins = [...PLUGINS];
+  if (fixture.isJSX) {
+    plugins.push('jsx');
+  }
+
+  let ast: unknown | 'ERROR' = 'ERROR';
```

Comment:
Suggestion: maybe use an enum for `'ERROR'` and `'NO ERROR'`? These constants are duplicated quite a few times.